Core/org.emftext.language.java/src/org/emftext/language/java/extensions/references/ReferenceExtension.java

```diff
@@ -101,6 +101,9 @@ else if (me instanceof ReflectiveClassReference) {
 		else if (me instanceof ElementReference) {
 			ReferenceableElement target = 
 				(ReferenceableElement) ((ElementReference) me).getTarget();
+			
+			if(target == null)
```

Comment:
@Adomat , can you please format the code correctly (whitespace after if and curly braces around the return statement)? Then I can merge this.
```suggestion
			if (target == null) {
				return null;
			}
```